benchmarking/nursery/benchmark_automl/benchmark_main.py

```diff
@@ -0,0 +1,148 @@
+from typing import Dict, Optional, List
+
+import numpy as np
+import itertools
+import logging
+from argparse import ArgumentParser
+from tqdm import tqdm
+
+from benchmarking.blackbox_repository import load
+from benchmarking.blackbox_repository.simulated_tabular_backend import BlackboxRepositoryBackend
+from benchmarking.nursery.benchmark_automl.baselines import MethodArguments, methods
+from benchmarking.nursery.benchmark_automl.benchmark_definitions import benchmark_definitions
+
+from syne_tune.backend.simulator_backend.simulator_callback import SimulatorCallback
+from syne_tune.optimizer.schedulers.transfer_learning import TransferLearningTaskEvaluations
+from syne_tune.stopping_criterion import StoppingCriterion
+from syne_tune.tuner import Tuner
+from coolname import generate_slug
+
+
+def get_transfer_learning_evaluations(
+        blackbox_name: str,
```

Comment:
in the future it would be great to have a argument that allows to specify the sampling strategy (e.g 'all', 'random', 'BO') to select the transfer learning data

Reply:
Agree, currently, we only have the option to take everything or to subsample at random.